crates/stackable-operator/src/commons/authentication/kerberos.rs

```diff
@@ -0,0 +1,12 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(
+    Clone, Debug, Deserialize, Eq, Hash, JsonSchema, Ord, PartialEq, PartialOrd, Serialize,
+)]
+#[serde(rename_all = "camelCase")]
+pub struct AuthenticationProvider {
+    /// Mandatory secret class used for producing keytabs.
```

Comment:
Not all backends produce a keytab, e.g. the k8sSearch backend uses an existing one the user provided via a Secret
```suggestion
    /// Mandatory SecretClass used to obtain keytabs.
```

Reply:
Thanks for clarifying!